server/odc-service/src/main/java/com/oceanbase/odc/service/task/config/TaskFrameworkConfiguration.java

```diff
@@ -84,6 +86,9 @@ public SchedulerFactoryBean taskFrameworkSchedulerFactoryBean(DataSource dataSou
         schedulerFactoryBean.setSchedulerName(taskFrameworkSchedulerName);
         schedulerFactoryBean.setStartupDelay(taskFrameworkProperties.getQuartzStartDelaySeconds());
         schedulerFactoryBean.setTaskExecutor(executor);
+        Properties properties = new Properties();
+        properties.put("org.quartz.jobStore.class", ResourceDetectJobStore.class.getName());
```

Comment:
magic value

Reply:
is it a standard method to set custom JobStore?

Reply:
yes,  it's standard method,  reference `SchedulerFactoryBean` line 575 ,
`mergedProps.putIfAbsent(StdSchedulerFactory.PROP_JOB_STORE_CLASS, LocalDataSourceJobStore.class.getName());`
but the magic value can be replaced by constants